tests/Google/AdsApi/AdWords/Reporting/v201809/RequestOptionsFactoryTest.php

```diff
@@ -418,11 +418,20 @@ private function getFormattedUserAgent()
 
     private function formatGuzzleInfo()
     {
-        $guzzleInfoTokens = ['GuzzleHttp/' . ClientInterface::VERSION];
+        $guzzleInfoTokens = ['GuzzleHttp/' . $this->getGuzzleVersion()];
```

Comment:
Please remove the new function and simply use:
        $guzzleInfoTokens = ['GuzzleHttp/' . ClientInterface::MAJOR_VERSION];